src/Practitioner/Settings/index.js

```diff
@@ -1,215 +1,289 @@
-        paddingTop: "1em",
-        borderRight: "solid 1px lightgray"
-    },
-    navItemContainer: {
-        listStyle: "none",
-        height: "2.5em"
-    },
-    navItem: {
-        boxSizing: "border-box",
-        height: "100%",
-        margin: ".5em 0 0 0",
-        padding: ".25em",
-        display: "flex",
-        width: "90%",
-        borderRadius: "7px",
-        color: props => props.selected ? "white" : Colors.buttonBlue,
-        backgroundColor: props => props.selected ? Colors.textGray : Colors.lightgray,
-        "& > span:first-letter": {
-            textTransform: "capitalize"
-        },
-        alignItems: "center",
-        justifyContent: "flex-start",
-        "& > *:first-child": {
-            marginRight: "5px"
-        },
-        fontSize: "1em"
+  mobileNavLinks: {
+    textDecoration: 'none',
+  },
 
+  navItemContainer: {
+    listStyle: 'none',
+    width: '100%',
+    height: '2.5em',
+  },
+  navItem: {
+    boxSizing: 'border-box',
+    height: '100%',
+    margin: '1em 0 0 0',
+    padding: '1.5em 1.2em',
+    display: 'flex',
+    width: '100%',
+    borderRadius: '7px',
+    color: (props) => (props.selected ? 'white' : Colors.buttonBlue),
+    backgroundColor: (props) =>
+      props.selected ? Colors.textGray : Colors.lightgray,
+    '& > span:first-letter': {
+      textTransform: 'capitalize',
     },
-    header: {
-        fontSize: "1.5em",
-    },
-    list: {
-        padding: "1em 0 0 0",
-        margin: 0
-    },
-    body: {
-        padding: "1em",
-        flex: 1,
-        height: "100vh",
-        overflow: "scroll",
-        boxSizing: "border-box"
-    },
-    bodyContent: {
-        borderTop: "1px solid black",
-        marginTop: "1em",
-        width: "100%"
+    alignItems: 'center',
+    justifyContent: 'flex-start',
+    '& > *:first-child': {
+      marginRight: '5px',
     },
-    patientInformation: {
-        width: "50%"
-    },
-    topInfo: {
-        padding: "1em"
-    },
-    password: {
-        width: "50%"
-    }
-})
+    fontSize: '1em',
+  },
+  header: {
+    fontSize: '1.5em',
+  },
+  list: {
+    padding: '1em 0 0 0',
+    margin: 0,
+  },
+  body: {
+    padding: '1em',
+    flex: 1,
+    height: '100vh',
+    overflow: 'scroll',
+    boxSizing: 'border-box',
+  },
+  bodyContent: {
+    borderTop: '1px solid black',
+    marginTop: '1em',
+    width: '100%',
+  },
+  patientInformation: {
+    width: '50%',
+  },
+  topInfo: {
+    padding: '1em',
+  },
+  password: {
+    width: '50%',
+  },
+});
 
 const Settings = (props) => {
+  const classes = useStyles();
+  const { isMobile } = useWindowSize();
+  const location = useLocation();
 
-    const classes = useStyles();
-    const { practitionerUIStore } = useStores();
+  return (
+    <div>
+      {isMobile ? (
+        <>{location.pathname === '/settings' ? <SettingsNav /> : <Routes />}</>
+      ) : (
+        <div className={classes.report}>
+          <SettingsNav className={classes.body} />
```

Comment:
SettingsNav doesnt take any props so this className prop isnt doing anything

Reply:
Absolutely.

Reply:
resolved